source/guides/network/running-a-symbol-node-manually.rst

```diff
@@ -0,0 +1,194 @@
+.. post:: 05 Mar, 2021
+    :category: Network
+    :excerpt: 1
+    :nocomments:
+
+##############################
+Running a Symbol node manually
+##############################
+
+This guide walks you through the process of setting up a node to join |codename|'s network **without** using Symbol Bootstrap.
+
+|symbol-bootstrap| is a very handy tool that allows the **quick deployment** of |codename| nodes. However, it relies on `Docker <http://docker.com>`__ technology which might not be available (or allowed) in all scenarios. This guide takes the longer route and explains how to setup a |codename| node **manually**. If you would like to use Symbol Bootstrap use :doc:`this guide <running-a-symbol-node>` instead.
+
+The next sections explain first how to build a :ref:`peer-node` and then expand it to convert it into an :ref:`api-node`.
+
+.. note:: The following instructions have only been verified to work on Linux.
+
+*****************
+Build a Peer node
+*****************
+
+Build the server
+================
+
+Follow the `Catapult server <https://github.com/nemtech/catapult-server>`__ instructions to **build** its binaries, either `manually <https://github.com/nemtech/catapult-server/blob/main/docs/BUILD-manual.md>`__ or using `Conan <https://github.com/nemtech/catapult-server/blob/main/docs/BUILD-conan.md>`__.
+
+Run the server
+==============
+
+**Download** the network configuration file from **A LOCATION YET TO BE DETERMINED**.
+
+Follow Catapult server's `instructions to run a node <https://github.com/nemtech/catapult-server/blob/main/docs/RUNPEERLIN.md>`__ using the downloaded network configuration.
+
+*****************
+Build an API node
+*****************
+
+Once you have the Catapult Server running, you can add **extra services** to turn it into an API node:
+
+.. figure:: ../../resources/images/diagrams/rest-detail.png
+    :align: center
+
+As shown in the diagram above, besides the **Catapult Server** you will also need:
+
+- A **Database** holding the REST data.
+- A **Catapult Broker** serializing accesses to the database.
+- A **REST gateway** accepting client requests and turning them into server commands or database queries.
+
+The following sections explain how to install each service. You will need all of them.
+
+Move to the ``_build`` folder created while building the server and create any additional folders requested in the next steps inside ``_build``.
+
+Run the database
+=====================
+
+1. **Install** `MongoDB <http://mongodb.com>`__ (at least version 4.4).
+
+   It is recommended to follow the `program's own installation instructions <https://docs.mongodb.com/manual/administration/install-community/>`__, and NOT install from your distro's packages.
+
+2. Create a directory to store the data files and **run the database**:
+
+   .. code-block:: bash
+
+      mkdir dbfiles
+      mongod --dbpath=dbfiles --wiredTigerCacheSizeGb 2 --bind_ip 127.0.0.1
```

Comment:
One of the options is not recognized in my system:
```bash
marc-os@roston:~/mm-s/main/catapult-server/_build/node_7900$ mongod --dbpath=dbfiles --wiredTigerCacheSizeGb 2 --bind_ip 127.0.0.1
Error parsing command line: unrecognised option '--wiredTigerCacheSizeGb'
try 'mongod --help' for more information

marc-os@roston:~/mm-s/main/catapult-server/_build/node_7900$ mongod --version
db version v4.4.4
Build Info: {
    "version": "4.4.4",
    "gitVersion": "8db30a63db1a9d84bdcad0c83369623f708e0397",
    "openSSLVersion": "OpenSSL 1.1.1j  16 Feb 2021",
    "modules": [],
    "allocator": "tcmalloc",
    "environment": {
        "distmod": "debian10",
        "distarch": "x86_64",
        "target_arch": "x86_64"
    }
}
```

Reply:
the offendign option is `--wiredTigerCacheSizeGb 2`. Without it, the process starts correctly (apparently).

Reply:
Yeah, the right option is `--wiredTigerCacheSizeGB`